cmake/modules/CTags.cmake

```diff
@@ -15,7 +15,7 @@ function(add_tags name)
       RESULT_VARIABLE result_code
       OUTPUT_VARIABLE submodules
       OUTPUT_STRIP_TRAILING_WHITESPACE)
-    if(${result_code} EQUAL 0)
+    if((${result_code} EQUAL 0) AND submodules)
```

Comment:
i think a better way is to find `git` before trying to use it:
```cmake
find_package(Git)
if(TAGS_EXCLUDE_OPTS)
  # ...
elseif(Git_FOUND)
  # exclude the submodules under SRC_DIR by default
  # ...
endif()
```

Reply:
sounds good